### ISSUE TITLE
Generate POM for a runtime

### ISSUE THREAD

**Issue by jvanzyl** [jvanzyl/provisio#13]

**Comment by mosabua** [comment]
@nineinchnick is the goal you created to populate the pom suitable also for a runtime setup?

**Comment by nineinchnick** [comment]
I did add the new goals to the Readme. Please check if the description there is enough. I think this issue could be closed now.

**Comment by nineinchnick** [comment]
Yes, that's what I meant. What do you mean by runtime? How to include this goal in the runtime scope? The installed/published POM is updated by default. We might add an example how to run the `validateDependencies` goal. BTW there's a typo in the README, it's mentioned as `verifyDependencies` which is incorrect.
